src/components/Map/index.tsx

```diff
@@ -19,107 +20,44 @@ interface AirQuality {
   ultraFineDustScale: number;
 }
 
-const INIT_ZOOM_LEVEL = 5;
 const MAX_ZOOM_LEVEL = 8;
 
 const Map = () => {
   const mapRef = useRef<HTMLDivElement | null>(null);
-  const dustInfoMarkers: kakao.maps.CustomOverlay[] = [];
-  const [map, setMap] = useState<kakao.maps.Map | null>(null);
-  const [location, setLocation] = useState(INIT_LOCATION);
-  const [zoomLevel, setZoomLevel] = useState(INIT_ZOOM_LEVEL);
-  const [currentCity, setCurrentCity] = useState('서울');
-
-  useEffect(() => {
-    kakao.maps.load(() => {
-      if (!mapRef.current) return;
-
-      const options = {
-        center: new kakao.maps.LatLng(location.latitude, location.longitude),
-        level: INIT_ZOOM_LEVEL,
-      };
-
-      const kakaoMap = new kakao.maps.Map(mapRef.current, options);
-
-      kakaoMap.setCenter(
-        new kakao.maps.LatLng(location.latitude, location.longitude)
-      );
-
-      navigator.geolocation &&
-        navigator.geolocation.getCurrentPosition((position) => {
-          setLocation(position.coords);
-
-          const { latitude, longitude } = position.coords;
-
-          const currentLocationMarker = new kakao.maps.CustomOverlay({
-            position: new kakao.maps.LatLng(latitude, longitude),
-            content: `<div class="circle-marker" />`,
-          });
-
-          currentLocationMarker.setMap(kakaoMap);
-
-          kakaoMap.setCenter(new kakao.maps.LatLng(latitude, longitude));
-        });
-
-      kakaoMap.setMaxLevel(MAX_ZOOM_LEVEL);
-
-      setMap(kakaoMap);
-    });
-  }, []);
-
-  useEffect(() => {
-    kakao.maps.load(() => {
-      if (!map) return;
-
-      kakao.maps.event.addListener(map, 'zoom_changed', () => {
-        const zoomLevel = map.getLevel();
-        setZoomLevel(zoomLevel);
-      });
-
-      const geocoder = new kakao.maps.services.Geocoder();
-
-      kakao.maps.event.addListener(map, 'dragend', function () {
-        const coords = map.getCenter();
-
-        geocoder.coord2Address(
-          coords.getLng(),
-          coords.getLat(),
-          (result, status) => {
-            if (status === kakao.maps.services.Status.OK) {
-              setCurrentCity(result[0].address.address_name.split(' ')[0]);
-            }
-          }
-        );
-      });
-    });
-  }, [map]);
-
-  const { data: airQuality, isLoading } = useQuery(
+  const dustInfoMarkersBySido: kakao.maps.CustomOverlay[] = [];
+  const dustInfoMarkersByCity: kakao.maps.CustomOverlay[] = [];
```

Comment:
```suggestion
  const sidodustInfoMarkers: kakao.maps.CustomOverlay[] = [];
  const citydustInfoMarkers: kakao.maps.CustomOverlay[] = [];
```
이렇게 쓰는게 더 좋을 것 같아요오.
by를 쓰니까 끝까지 읽어야 하는 피로감...이 있는 것 같아요!
저도 이전에 쓴 거 다 바꿔놓음..ㅎㅎ